packages/cubejs-backend-cloud/src/deploy.ts

```diff
@@ -43,7 +43,7 @@ export class DeployDirectory {
     }
 
     // blacklist
-    if (['dashboard-app', 'node_modules'].includes(baseName)) {
+    if (['.git', 'dashboard-app', 'node_modules'].includes(baseName)) {
```

Comment:
It already ignores all names started with by `return baseName.charAt(0) !== '.';`.